tutor/resources/styles/components/top-nav-bar.less

```diff
@@ -37,9 +37,18 @@
   padding-right: @navbar-padding-horizontal;
   padding-left:  @navbar-padding-horizontal;
 
+  .dropdown.open > a {
+      box-shadow: none;
+  }
+
   a {
     color: #777;
     text-decoration: none;
+
+    &:hover,
+    &[aria-expanded="true"] {
+      color: #333;
+    }
   }
 
```

Comment:
I think the dropdown rules could be moved to the `dropdown-menu` file that's included at the bottom, that way all menu style rules are consolidated.